spec/import_spec.rb

```diff
@@ -268,12 +302,14 @@ def upload_file!(name, ext='csv')
 
 
     context "with callback procs options" do
-      let(:options) { {
-          before_import: proc { |_|},
-          after_import: proc { |_|},
-          before_batch_import: proc { |_|},
-          after_batch_import: proc { |_|}
-      } }
+      let(:options) do
+        {
+            before_import: proc { |_|},
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Space missing inside }.